data/Record.js

```diff
@@ -67,6 +67,14 @@ export class Record {
         });
     }
 
+    /**
+     * Return an object containing the current values of all fields in this record.
+     * @returns {Object}
+     */
+    getData() {
```

Comment:
This seemed like a generally useful method to have for Records, and something we needed for setting up the clone data, so added this here

Reply:
Yes, although does really suggest we should have stored the data in its own bundle from the get go,

Reply:
Yes - filed #998 to track / review.  

We have discussed how it is disquieting that we mix in the record data fields as top-level properties - at the least it necessitated https://github.com/exhi/hoist-react/blob/395e285e9281373aad81574e1d6dca5c302fa24b/data/Record.js#L18

Reply:
Would we prefer to remove this new method for now in favor of just doing this work in RestGridModel (where it is also going to need to omit non-editable properties anyways)? Probably best to avoid adding this new method if we are going to provide a different interface for retrieving the data in a future release

Reply:
Yes I would support that change, for exactly those reasons.

Reply:
great point john H, and I agree.  Looks like this still needs to be done